runbotics/runbotics-scheduler/src/scheduler-database/process-output/process-output.entity.ts

```diff
@@ -0,0 +1,9 @@
+import { ProcessOutputType } from 'runbotics-common';
+import { Entity, PrimaryColumn } from 'typeorm';
+
+
+@Entity({ name: 'process_output' })
+export class ProcessOutput {
+    @PrimaryColumn({ type: 'varchar', length: 50 })
```

Comment:
shouldn't be `type: 'enum', enum: 'ProcessOutputType'` here?

Reply:
nvm in case my suggestions cause brake of current legacy java logic

Reply:
enum may cause the need for new migration each time it changes in the codebase. Unless it's not planned to change or is planned to change hardly ever I wouldn't add this